specification/paths/Returns-v1-returns-return_id-request-payment.json

```
@@ -0,0 +1,62 @@
+{
+  "parameters": [
+    {
+      "$ref": "#/components/parameters/path-return_id"
+    }
+  ],
+  "post": {
+    "tags": [
+      "ReturnPaymentRequests"
```

Since this endpoint starts with `returns/v1/returns/{return_id}` I think we can place it under `Returns`
```suggestion
      "Returns"
```

---

done